src/assets/style/global.less

```diff
@@ -508,3 +508,7 @@ body {
   font-style: normal;
   src: local('Gilroy'), url('../fonts/Gilroy-ExtraBold.otf');
 }
+
+.arco-drawer {
```

Comment:
Better not use a global css. You could add a class for your drawer and set the border under the class.